src/services/DocumentSelectionRequestProcessor.ts

```diff
@@ -156,6 +156,19 @@ export class DocumentSelectionRequestProcessor {
   				throw new AppError(HttpCodesEnum.SERVER_ERROR, "Unknown document type");
   		}
 
+  		try {
+  			this.logger.info("Updating documentType in Session Table: ", { docType });
```

Comment:
[nit] we seem to used a combination of `documentType`, `documentUsed` and `selected document` could we just make sure these are consistent please?